### PR TITLE
automation, Align sig-operator in automation scripts

### PR DESCRIPTION
**Pull request by oshoval**
Align sig-operator as the other sigs in
automation/test.sh script.

This way it can set TARGET as
k8s-1.x-sig-operator.

Signed-off-by: Or Shoval <oshoval@redhat.com>

```release-note
None
```

### CODE REVIEW

**Comment by oshoval** [comment]
@fgimenez can you take a look on this one please ?
prerequisites of https://github.com/kubevirt/project-infra/pull/1547

**Review by dhiller — APPROVED** [review]
/approve
/hold

Waiting for another reviewer

**Review by fgimenez — APPROVED** [review]
/lgtm

Thanks for taking care of this!

**Comment by oshoval** [comment]
> /approve
> /hold
> 
> Waiting for another reviewer

Lets remove the hold ? Federico reviewed

thanks

**Pull request merged**
